skbio/stats/distance/_base.py

```diff
@@ -685,6 +689,11 @@ def _is_id_pair(self, index):
                 len(index) == 2 and
                 all(map(lambda e: isinstance(e, string_types), index)))
 
+    def _find_duplicates(self, iterable):
```

Comment:
We recently wrote a function to do this in Qiita, some profiling using IPython seems to show it being faster than this implementation ... just sayin'

```python
In [27]: %timeit h = _find_duplicates(x)
1000 loops, best of 3: 368 µs per loop

In [28]: h
Out[28]: {0, 1, 2, 3, 4, 5, 6, 7, 8, 9}

In [29]: %timeit h = find_repeated(x)
10000 loops, best of 3: 147 µs per loop

In [30]: h
Out[30]: {0, 1, 2, 3, 4, 5, 6, 7, 8, 9}
```

[Code in Qiita](https://github.com/biocore/qiita/blob/01225be1227d788173b43cf6c1b047cc2076bfeb/qiita_db/util.py#L901).

If we put this in scikit-bio, we could import it in Qiita :neckbeard: